acme_tiny.py

```diff
@@ -188,11 +188,19 @@ def main(argv):
     parser.add_argument("--acme-dir", required=True, help="path to the .well-known/acme-challenge/ directory")
     parser.add_argument("--quiet", action="store_const", const=logging.ERROR, help="suppress output except for errors")
     parser.add_argument("--ca", default=DEFAULT_CA, help="certificate authority, default is Let's Encrypt")
+    parser.add_argument("--output", "-o", metavar="FILE", default=None, help="output file, default is standard output")
 
     args = parser.parse_args(argv)
     LOGGER.setLevel(args.quiet or LOGGER.level)
     signed_crt = get_crt(args.account_key, args.csr, args.acme_dir, log=LOGGER, CA=args.ca)
-    sys.stdout.write(signed_crt)
+    if args.output:
+        dirname, basename = os.path.split(args.output)
+        with tempfile.NamedTemporaryFile(prefix=basename, dir=dirname, delete=False) as f:
```

Comment:
What if the directory is not writable for the user running acme-tiny?
The final file might be writable, but not the directory itself.